code/_helpers/text.dm

```diff
@@ -355,7 +355,7 @@
 		return tagdesc
 	if(!text_tag_cache[tagname])
 		var/icon/tag = icon(text_tag_icons, tagname)
-		text_tag_cache[tagname] = bicon(tag, TRUE, "text_tag")
+		text_tag_cache[tagname] = icon2html(tag, C, extra_classes = "text_tag")
```

Comment:
OOC, DCHAT, LOOC.... icons aren't visible ingame, i suspect it has something to do with this.